### DISCUSSION TITLE
How to see abandoned cart in Saleor Dashboard ?

### DISCUSSION THREAD

**Discussion by neerajgupta2407** [mirumee/saleor-dashboard#3590] [General] [answered]
HI 
I want to know if a user abandons the cart, does not complete the payment, 
Is there any way to see it on Saleor Dashboard ?

**Comment by timuric** [comment, marked as answer]
Hi @neerajgupta2407, you could create an app to browse checkouts https://github.com/saleor/saleor-app-template
